<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auth</class>
 <widget class="QWidget" name="Auth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickDialog</string>
  </property>
  <property name="windowIcon">
   <iconset theme="contact-new"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(24, 0, 36, 255), stop:0.312849 rgba(38, 8, 72, 255), stop:0.731844 rgba(44, 18, 97, 255), stop:1 rgba(50, 29, 125, 255));</string>
  </property>
  <widget class="QWidget" name="widget_down" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pushButton_Auth">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вход в аккаунт с введёнными учётными данными&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 0, 52, 255), stop:0.312849 rgba(38, 8, 72, 255), stop:0.731844 rgba(44, 18, 97, 255), stop:1 rgba(50, 29, 125, 255));
border: 1px solid rgb(0, 0, 127);
color: rgb(237, 253, 255);
font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Registr">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Регистрация нового аккаунта по введённым учётным данным&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(34, 0, 52, 255), stop:0.312849 rgba(38, 8, 72, 255), stop:0.731844 rgba(44, 18, 97, 255), stop:1 rgba(50, 29, 125, 255));
border: 1px solid rgb(0, 0, 127);
color: rgb(237, 253, 255);
font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Auth">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>347</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 42pt &quot;Comic Sans MS&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(239, 255, 246);</string>
    </property>
    <property name="text">
     <string>Авторизация</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>501</width>
      <height>115</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_Username">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 0, 83);
color: rgb(255, 255, 255);
border: 2px solid rgb(212, 255, 254);
border-top: 0px;
border-left: 0px;
border-right: 0px;</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>Имя пользователя</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Password">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(46, 0, 83);
color: rgb(255, 255, 255);
border: 2px solid rgb(212, 255, 254);
border-top: 0px;
border-left: 0px;
border-right: 0px;</string>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_Error">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>441</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>441</width>
      <height>27</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>441</width>
      <height>27</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(29, 119, 255, 0);
color: rgb(255, 133, 135);
font: italic 14pt &quot;Comic Sans MS&quot;;

</string>
    </property>
    <property name="text">
     <string>Пользователь с таким именем уже существует</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
